### ISSUE TITLE
Desktop - Blue outline on top of the screen

### ISSUE THREAD

**Issue by pedromvpg** [BlueWallet/BlueWallet#1473]
Focused/outline at the top of the UI won't go away even after interacting with other element

<img width="901" alt="Screen Shot 2020-08-08 at 12 37 29 AM" src="https://user-images.githubusercontent.com/807505/89910427-3a80a500-dbbe-11ea-897e-e5cdb523c17f.png">

<img width="972" alt="Screen Shot 2020-08-08 at 12 38 48 AM" src="https://user-images.githubusercontent.com/807505/89910423-38b6e180-dbbe-11ea-9d2b-2d993c2c0bdf.png">

**Comment by ncoelho** [comment]
aha  :) 

ACK

**Comment by marcosrdz** [comment]
I'm trying to replicate this. Any chance you have VoiceOver enabled? This appears like the voiceover "focus" box.

**Comment by pedromvpg** [comment]
No, but selecting another app and coming back to bluewallet removed it.

<img width="1681" alt="Screen Shot 2020-08-11 at 7 28 50 PM" src="https://user-images.githubusercontent.com/807505/89958852-f961b280-dc08-11ea-92a0-8bc6e258bd54.png">


![Aug-11-2020 19-29-51](https://user-images.githubusercontent.com/807505/89958881-11393680-dc09-11ea-85f4-a3f365905741.gif)

**Comment by Overtorment** [comment]
is it still happening?
